wordpress/var/www/html/wp-config.php.ctmpl

```diff
@@ -12,10 +12,11 @@
  * The following database info is needed here, but is ignored after we load HyperDB.
  * See db-config.php.ctmpl for details.
  */
-define( 'DB_NAME',     '{{key "wordpress/config/DB_NAME"}}' );
-define( 'DB_USER',     '{{key "wordpress/config/DB_USER"}}' );
-define( 'DB_PASSWORD', '{{key "wordpress/config/DB_PASSWORD"}}' );
-define( 'DB_HOST',     '{{with index (service "dbmaster") 0}}{{.Address}}:{{.Port}}{{end}}' );
+ ## this is a test
+define( 'DB_NAME',     '{{env "MYSQL_DATABASE"}}' );
+define( 'DB_USER',     '{{env "MYSQL_USER"}}' );
+define( 'DB_PASSWORD', '{{env "MYSQL_PASSWORD"}}' );
+define( 'DB_HOST',     '{{range service "mysql-primary"}}{{.Address}}:{{.Port}}{{end}}' );
```

Comment:
FYI, the HyperDB template I'd started: https://github.com/10up/triton-wordpress/blob/wordpress/wordpress/var/www/html/db-config.php.ctmpl#L23-L70